### DISCUSSION TITLE
Hi, is there any extensions for display dependencies references?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#575] [Extension Development QnA]
in nodejs package.json, 

how to know the packages references ? is there any extensions to do it?

![image](https://user-images.githubusercontent.com/12727582/233521832-2ff6dcf3-cf7e-4625-8850-cff0243b9116.png)

**Comment by digitarald** [comment]
As these discussions are meant for extension development, I recommend posting this one to stackoverflow.
